### PR TITLE
Updates project to work with Swift 5.0/Xcode 10.2

### PR DESCRIPTION
**Pull request by aaronbrethorst**

### CODE REVIEW

File: PMKMapKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -305,7 +307,7 @@
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos macosx appletvsimulator appletvos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 5.0;
```
Comment (mxcl) [inline PMKMapKit.xcodeproj/project.pbxproj]: Needs to be 4.0.

File: PMKMapKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -365,7 +368,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = org.promisekit.MapKit;
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos macosx appletvsimulator appletvos";
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 5.0;
```
Comment (mxcl) [inline PMKMapKit.xcodeproj/project.pbxproj]: Needs to be 4.0.

File: PMKMapKit.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -418,7 +421,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_SUPPRESS_WARNINGS = YES;
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 5.0;
```
Comment (mxcl) [inline PMKMapKit.xcodeproj/project.pbxproj]: Needs to be 4.0.

File: PMKMapKit.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -431,7 +434,7 @@
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_SUPPRESS_WARNINGS = YES;
-				SWIFT_VERSION = 4.0;
```
Comment (mxcl) [inline PMKMapKit.xcodeproj/project.pbxproj]: Needs to be 4.0.

File: Tests/TestMapKit.swift [hunk 5]
```diff
@@ -8,12 +8,12 @@ class Test_MKDirections_Swift: XCTestCase {
...
         class MockDirections: MKDirections {
-            override func calculate(completionHandler: @escaping MKDirectionsHandler) {
-                completionHandler(MKDirectionsResponse(), nil)
+            override func calculate(completionHandler: @escaping MKDirections.DirectionsHandler) {
+                completionHandler(MKDirections.Response(), nil)
             }
         }
 
-        let rq = MKDirectionsRequest()
+        let rq = MKDirections.Request()
```
Comment (mxcl) [inline Tests/TestMapKit.swift]: This probably needs to have a `#if swift` to maintain compatibility with older Swifts.

**Review by mxcl — CHANGES_REQUESTED** [review]

**Comment by aaronbrethorst** [comment]
@mxcl thanks for the feedback. I made the requested changes.

**Comment by mxcl** [comment]
Unfortunately we have revlock, we need CocoaPods 1.7.0-beta now to support the new attribute `swift_versions`. I will merge, this is fine IMO.

**Pull request merged**